examples/compiler/class_table.rs

```diff
@@ -39,6 +39,7 @@ fn all_fields(db: &impl ClassTableDatabase, (): ()) -> Arc<Vec<DefId>> {
             .flat_map(|def_id| {
                 let fields = db.fields(def_id);
                 (0..fields.len()).map(move |i| fields[i])
-            }).collect(),
+            })
+            .collect(),
```

Comment:
btw, I should add that I always edit the salsa code with rustfmt-on-save enabled; I would appreciate it if you did the same. (Not sure if this edit came from rustfmt? Could be, if we are running slightly different versions...)

Reply:
Yep I have rustfmt-on-save enabled! This may have come from when I had the file open for understanding the example; I won't push single file formatting changes like this in the future.

As for the versioning, I was originally running this in nightly as I mentioned. I'm not sure if that would make the difference, but it could explain it.